powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/SignatureServiceBehavior.java

```diff
@@ -140,10 +139,12 @@ public VerifyOfflineSignatureResponse verifyOfflineSignature(String activationId
                                                                  String dataString, CryptoProviderUtil keyConversionUtilities)
             throws UnsupportedEncodingException, InvalidKeySpecException, InvalidKeyException {
 
-        final VerifySignatureResponse verifySignatureResponse = verifySignature(activationId, signatureType, signature, dataString, null, keyConversionUtilities, true);
+        KeyValueMap additionalInfo = new KeyValueMap();
```

Comment:
We should probably allow `additionalInfo` to be `null`, so that we do not need to create dummy objects whenever calling the method. We could do null check inside `verifySignature`...

Reply:
Ok, I will update it.